src/components/Logo/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import configuration from 'shared/services/configuration/configuration';
 
 export const Logo = props => (
-  <a href={configuration.links.home}>
-    <img alt="Logo" src={configuration.logoUrl} height={configuration.logoHeight} {...props} />
+  <a href={props.tall ? configuration.links.home : '/'}>
+    <img
+      alt="Logo"
+      src={configuration.logoUrl}
+      width={props.tall ? configuration.logoWidth : configuration.logoWidthSmall}
+      height={props.tall ? configuration.logoHeight : configuration.logoHeightSmall}
+      {...props}
+    />
   </a>
```

Comment:
Do we want to use try/catch blocks everywhere we use the configuration?

Reply:
Maybe not. Conditional chaining will also do.

Reply:
All configuration properties are set with a default value in `environment.conf.json` and should thus be available. In case a property is not available, something is wrong and the exception should be thrown.
On the other hand, we could have the configuration service treat a property that is not set, as being set with `null`. No need for exceptions at all.